Explorer/Assets/DCL/SDKComponents/SceneUI/Systems/UITransform/UITransformParentingSystem.cs

```diff
@@ -43,7 +43,14 @@ private void OrphanChildrenOfDeletedEntity(ref UITransformComponent uiTransformC
 
             foreach (EntityReference childEntity in children)
             {
-                ref var uiTransform = ref World.Get<UITransformComponent>(childEntity.Entity);
+                ref UITransformComponent uiTransform = ref World.TryGetRef<UITransformComponent>(childEntity.Entity, out bool exists);
+
+                if (!exists)
+                {
+                    ReportHub.LogError(GetReportCategory(), $"Trying to remove an ${nameof(UITransformComponent)}'s child but no component has been found on entity {childEntity.Entity}");
```

Comment:
I would change the error description to something like: "Trying to unparent an ${nameof(UITransformComponent)}'s child but no component has been found on entity {childEntity.Entity}"